### PR TITLE
SPARKC-130: Upgrade to Spark 1.2.2 and enable Java API for Scala 2.11

### PR DESCRIPTION
**Pull request by jacek-lewandowski**

### CODE REVIEW

File: project/Versions.scala [hunk 1]
```diff
@@ -17,7 +17,7 @@ import scala.util.Properties
  */
 object Versions {
 
-  val crossScala = Seq("2.11.5", "2.10.4")
+  val crossScala = Seq("2.11.6", "2.10.5")
```
Comment (helena) [inline project/Versions.scala]: We don't want this?

Seq("2.11.5", "2.11.6", "2.10.4", "2.10.5")

Comment (pkolaczk) [inline project/Versions.scala]: Why would you want 2.11.6 *and* 2.11.5 as well as 2.10.5 *and* 2.10.4? They are binary compatible.

Comment (helena) [inline project/Versions.scala]: We would not. After writing that I realized it would cause more travis builds. But I never deleted my above comment.

**Comment by pkolaczk** [comment]
+1

**Comment by helena** [comment]
+1

**Pull request merged**